6502 Tapeout HSRM

danielh186 opened the pull request.

danielh186 committed:
6502 Tapeout HSRM

KrzysztofHerman commented:
Dear Contributor,
since we are closing the submission please change the final GDS name adding the following prefix to the file
`FMD_QNC_18_`

danielh186 committed:
Change gds filename

KrzysztofHerman commented:
@danielh186 

I have verified your design using internal minimal DRC  It is clean and almost ready to be accepted. I would like to kindly ask you for the following modifications:

0)  Remove `235/4` green square covering your chip since it causes `Seal.l` DRCs

1) Due to internal issues we have to ask you to open the file in klayout and save it again using `File->Save As..` and provide the following options: 

![image](https://github.com/user-attachments/assets/9c03da59-d0fa-4c82-b2a2-366f080b93f8)

2) Please change the `gds` file name adding a prefix: `FMD_QNC_18`. 

3) Please provide us also some contact data and a shipping address. 

4) Please download the Hiring Agreement from [here](https://ihp-open-ip.readthedocs.io/en/latest/_downloads/c82c58426246cc84f3bbce282975b255/samples.rtf) and sign it. If you are with an university please ask for signature some representative: your advisor, group leader, departments head.  

5) The data mentioned in the points 3-4 you can send to the following [email](openpdk@ihp-microelectronics.com). Please mention this tape out name `TO_Apr2025` in the email title.

danielh186 committed:
Modify GDS as requested

KrzysztofHerman commented:
@danielh186 
Your seal ring is broken. Please fix and update.

danielh186 committed:
Fix sealring

Editing the design with klayout version 0.30.0 broke the sealring.
Rolling back to version 0.29.12 fixes the issue.

KrzysztofHerman commented:
@danielh186 
Congratulations!
The design was accepted for production:
```
File name: FMD_QNC_18_6502cpu.gds
Top cell name: cpu_top
Testfield: T586
Check Summe: 3fe2eec59e27f180cf0cbf0bebf130ef
Sent: 11-04-2025, 13:16:27
registered size: (1.97 mm X 0.88 mm ) 
calculated size: (1.906 mm X 0.816 mm))
ID (use this as reference for questions): 54573

```

Please do not change the GDS files anymore.
I will merge this PR to the main branch
